k8s/packer/Makefile

```diff
@@ -0,0 +1,9 @@
+testground-ami-images:
+	packer build -var 'source_ami=ami-093dda5e01f6102a8' -var 'aws_region=eu-central-1'   testground-ami.json
```

Comment:
@coryschwartz I know there is a way to copy AMIs cross-region, and that would be better than rebuilding them in every region, but I couldn't figure out quickly how to do that with the `awscli` - all tutorials were showing how to do it through the console. Definitely something worth fixing before we release.